exchange/utils.go

```diff
@@ -41,16 +42,27 @@ func cleanOpenRTBRequests(ctx context.Context,
 	// Clean PI from bidrequests if not allowed per GDPR
 	gdpr := extractGDPR(orig, usersyncIfAmbiguous)
 	consent := extractConsent(orig)
-	if gdpr == 1 {
-		for bidder, bidReq := range requestsByBidder {
-			// Fixes #820
+
+	for bidder, bidReq := range requestsByBidder {
+		var applyGDPR, applyCOPPA, isAMP bool
```

Comment:
Shouldn't `applyCOPPA` and `isAMP` be defined and set outside of the loop, as they don't depend on the bidder in question?

Reply:
Good point. I will change and update. Thanks!